.github/workflows/test.yml

```diff
@@ -0,0 +1,24 @@
+name: Test CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
```

Comment:
Please rename this to `test` since we're not building the library.

Reply:
✅